### ISSUE TITLE
Create Merge Request is not showing settings dialog when user did not set API information

### ISSUE THREAD

**Issue by Amertz08** [PavlikPolivka/GitLabProjects#18]
1. Create a new branch locally
1. Make changes to README.md
1. commit & push
1. Try and create merge request via Intellij

GitLab version 8.2.0
Intellij 14.1.4
```
null
java.lang.NullPointerException
	at java.util.regex.Matcher.getTextLength(Matcher.java:1283)
	at java.util.regex.Matcher.reset(Matcher.java:309)
	at java.util.regex.Matcher.<init>(Matcher.java:229)
	at java.util.regex.Pattern.matcher(Pattern.java:1093)
	at com.ppolivka.gitlabprojects.util.GitLabUtil.isGitLabUrl(GitLabUtil.java:97)
	at com.ppolivka.gitlabprojects.util.GitLabUtil.findGitLabRemote(GitLabUtil.java:80)
	at com.ppolivka.gitlabprojects.merge.GitLabMergeRequestWorker$Util.fillRequiredInfo(GitLabMergeRequestWorker.java:86)
	at com.ppolivka.gitlabprojects.merge.request.GitLabCreateMergeRequestWorker$3.convert(GitLabCreateMergeRequestWorker.java:158)
	at com.ppolivka.gitlabprojects.merge.request.GitLabCreateMergeRequestWorker$3.convert(GitLabCreateMergeRequestWorker.java:151)
	at com.ppolivka.gitlabprojects.util.GitLabUtil$2.run(GitLabUtil.java:189)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
	at com.intellij.openapi.progress.impl.CoreProgressManager$8.run(CoreProgressManager.java:367)
	at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:152)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:452)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:402)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:137)
	at com.intellij.openapi.application.impl.ApplicationImpl$10$1.run(ApplicationImpl.java:617)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:400)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
```

**Comment by Amertz08** [comment]
Realized I did not add my API key under the GitLab settings...probably could go ahead and close this. That or maybe catch the exception and notify the user?

**Comment by PavlikPolivka** [comment]
There should be a key setting dialog window opened when trying to do something without setting it.  

This is not working for merge requests for some reason. Leaving this open.

**Comment by PavlikPolivka** [comment]
Released as part of 1.3.0 release
